ui/public/src/connect.js

```diff
@@ -26,7 +26,7 @@ function linesToHTML(lines) {
   return lines
     .split('\n')
     .map(
-      l =>
```

Comment:
I think you must be using an old version of prettier. I'm using the latest in dapp-fungible-faucet. I think this shouldn't change

Reply:
I'm using the packages specified in the `yarn.lock`.  The new prettier prefers parens around single arguments.  In the commit before this one I copy-pasted some code and hadn't yet run lint-fix.